src/constants.cr

```diff
@@ -1,6 +1,11 @@
 module PassKit
-  SIGNING_CERT = ENV["SIGNING_CERT"]?
-  PRIVATE_KEY = ENV["PRIVATE_KEY"]?
-  PRIVATE_KEY_PASSWORD = ENV["PRIVATE_KEY_PASSWORD"]?
-  WWDR_CERT = ENV["WWDR_CERT"]?
+  SIGNING_CERT         = env_base64("SIGNING_CERT")
+  PRIVATE_KEY          = env_base64("PRIVATE_KEY")
+  WWDR_CERT            = env_base64("WWDR_CERT")
+  PRIVATE_KEY_PASSWORD = ENV["PRIVATE_KEY_PASSWORD"]
+
+  private def self.env_base64(key : String) : String
+    raise ArgumentError.new("Base64-encoded ENV #{key} is empty") unless ENV[key].presence
```

Comment:
Since this is a library, I would suggest exposing a configuration object.
[There's a handy library to do this](https://github.com/luckyframework/habitat).
Here's what we do in [hound-dog](https://github.com/place-labs/hound-dog/blob/master/src/hound-dog/settings.cr).



Additionally, here's a small update to prevent a key error.
```suggestion
    raise ArgumentError.new("Base64-encoded ENV #{key} is empty") unless ENV[key]?.presence
```